_data/events1.yml

```diff
@@ -0,0 +1,132 @@
+- title: |
+    Living on the Edge : Understanding Edge Computing on AWS
+  description: |
+    In this final talk of the series on Serverless Computing, we will focus on Edge Computing, understanding AWS Lambda@Edge and Amazon CloudFront Functions. You will learn how to choose between these tools, best practices for their use, and optimal scenarios for leveraging Edge Computing.
+  category: Online Event
+  category_name: Online Event

```

Comment:
this attribute was renamed to `category_style`: coding-club please have a look in the listed options.